site/content/en/docs/Reference/kustomize_cli.md

```diff
@@ -1,8 +1,77 @@
 ---
-title: "kustomize CLI"
-linkTitle: "kustomize CLI"
+title: "Command line tool (kustomize)"
+linkTitle: "Command line tool (kustomize)"
 weight: 2
 date: 2017-01-05
-description: >
-  Reference for the kustomize CLI
----
\ No newline at end of file
+description:
+---
+
+This overview covers `kustomize` syntax, describes the command operations, and provides common examples.
+
+## Syntax
+Use the following syntax to run `kustomize` commands from your terminal window:
+
+```bash
+kustomize [command]
+```
+
+The `command` flag specifies the operation that you want to perform, for example `create`, `build`, `cfg`.
+
+If you need help, run `kustomize help` from the terminal window.
+
+## Operations
+The following table includes short descriptions and the general syntax for all the `kustomize` operations.
+
+Operation | Syntax | Description
+--- | --- | ---
+build | kustomize build DIR [flags] | Build a kustomization target from a directory or URL.
+cfg | kustomize cfg [command] | Commands for reading and writing configuration.
+completion | kustomize completion [bash\|zsh\|fish\|powershell] | Generate shell completion script.
+create | kustomize create [flags] | Create a new kustomization in the current directory.
+edit | kustomize edit [command] |  Edits a kustomization file.
+fn | kustomize fn [command] | Commands for running functions against configuration.
+localize | kustomize localize [target [destination]] [flags] | [Alpha] Creates localized copy of target kustomization root at destination.
+version | kustomize version [flags] | Prints the kustomize version.
```

Comment:
How about doing it like this?

```suggestion
build | `kustomize build DIR [flags]` | Build a kustomization target from a directory or URL.
cfg | `kustomize cfg [command]` | Commands for reading and writing configuration.
completion | `kustomize completion [bash\|zsh\|fish\|powershell]` | Generate shell completion script.
create | `kustomize create [flags]` | Create a new kustomization in the current directory.
edit | `kustomize edit [command]` |  Edits a kustomization file.
fn | `kustomize fn [command]` | Commands for running functions against configuration.
localize | `kustomize localize [target [destination]] [flags]` | [Alpha] Creates localized copy of target kustomization root at destination.
version | `kustomize version [flags]` | Prints the kustomize version.
```

It would look like this:
<img width="775" alt="image" src="https://github.com/kubernetes-sigs/kustomize/assets/73531614/5648d40b-2f64-42a2-ba4f-75b683fb6198">

Reply:
Added code syntax to markdown.